tests/loginacademy.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+test('Academy login test one', async ({ page }) => {
+
+  await page.goto('https://rahulshettyacademy.com/client')
```

Comment:
You can move `https://rahulshettyacademy.com` to the playwright config file to the variable `baseUrl`. It's commented by default. So then you'll be able to call just:
```suggestion
await page.goto('/client')
```